recipes/boost/all/conanfile.py

```diff
@@ -672,8 +700,7 @@ def _build_bcp(self):
         with tools.vcvars(self.settings) if self._is_msvc else tools.no_op():
             with tools.chdir(folder):
                 command = "%s -j%s --abbreviate-paths toolset=%s" % (self._b2_exe, tools.cpu_count(), self._toolset)
-                if "debug_level" in self.options:
-                    command += " -d%d" % self.options.debug_level
+                command += " -d%d" % self.options.debug_level
```

Comment:
Thank you for finally fixing this 😄 https://github.com/conan-io/conan-center-index/pull/4121

Reply:
but it was already fixed in #4121, isn't it?

Reply:
Indeed.
This is https://github.com/conan-io/conan-center-index/pull/4121#discussion_r553544541

Reply:
It was my optional suggestion here, https://github.com/conan-io/conan-center-index/pull/4121#discussion_r553544541... just some clean up